template/markdownthemeistqb_common.sty

```diff
@@ -111,6 +111,12 @@
     books .tl_gset:N = \istqbbooksname,
     articles .tl_gset:N = \istqbarticlesname,
   }
+\keys_define:nn
+  { istqb / language / lists }
+  {
+    table .tl_gset:N = \listtablename,
+    figure .tl_gset:N = \listfigurename,
+  }
```

Comment:
We don't need to do anything special besides setting the standard LaTeX macros `\listtablename` and `\listfigurename`, which were previously only set for Czech, Slovak, and English but not for German.

Regardless, this PR sets the translations for all languages. This way, we are not dependent on the default translations from LaTeX, which may change unexpectedly and which may not be consistent with ISTQB's templates and expectations.